File: swift/Programming/Programming/loop/Loop.swift
//
//  foreach.swift
//  swift
//
//  Created by ke on 4/18/24.
//

import Foundation

class Loop {
    let games = ["Mahjong", "Gomoku Narabe Ren", "Popeye no Eigo Asobi"]
    
    func loop1(){
        for game in games {
            print(game)
        }
    }
    
    func loop2(){
        for case "Mahjong" in games {
            print("Mah")
        }
    }
    
    
    
}

File: go/loop/loop.go
package loop

import "fmt"

func loop1() {
	var pow = []int{1, 2, 4, 8, 16, 32, 64, 128}

	for i, v := range pow {
		fmt.Printf("2**%d = %d\n", i, v)
	}
}

func loop2() {
	for i := range 10 {
		fmt.Println(i)
	}
}
